.github/workflows/post-release.yml

```diff
@@ -23,31 +23,32 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up Python 3.8
+    - name: Set up Python 3.10
       uses: actions/setup-python@v2
       with:
-        python-version: '3.8'
+        python-version: '3.10'
 
     - name: Install python dependencies
       run: |
-        pip install --upgrade pip
-        pip install transifex-client sphinx-intl
-        pip install -e .[docs,tests,rest,atomic_tools]
+        pip install -U -e .[docs,tests,rest,atomic_tools]
 
     - name: Build pot files
       env:
         READTHEDOCS: 'True'
-        RUN_APIDOC: 'False'
+        RUN_APIDOC: 'True'
       run:
-        sphinx-build -b gettext docs/source locale
+        make -C docs gettext
 
     - name: Setting transifex configuration and upload pot files
       env:
         PROJECT_NAME: aiida-core
         USER: ${{ secrets.TRANSIFEX_USER }}
         PASSWD: ${{ secrets.TRANSIFEX_PASSWORD }}
       run: |
-        tx init --no-interactive
-        sphinx-intl update-txconfig-resources --pot-dir locale --transifex-project-name ${PROJECT_NAME}
+        sphinx-intl create-txconfig
+        sphinx-intl update-txconfig-resources --pot-dir docs/build/locale --transifex-project-name ${PROJECT_NAME}
         echo $'[https://www.transifex.com]\nhostname = https://www.transifex.com\nusername = '"${USER}"$'\npassword = '"${PASSWD}"$'\n' > ~/.transifexrc
-        tx push -s
+
+    - name: Push to transifex
+      run: |
+        tx push -t -s
```

Comment:
```suggestion
      run: tx push -t -s
```

Reply:
It is installed by `sphinx-intl[transifex]` in previous comment.

Reply:
However, I just found out they deprecate this extra requirement (https://github.com/sphinx-doc/sphinx-intl/pull/62) in the new version which is not released yet.
I can add a comment so can quickly update the CI workflow in the future.

Reply:
and maybe open an issue for it already linking to that PR, so you don't have to dig it out again next time